interview-questions.md

```diff
@@ -11,57 +11,83 @@ Then, research the question to expand on your answer. Even if you feel you have
 
   Your answer:
 
-  Researched answer:
+  The 'this' in JavaScript is referring to a part of itself. We see 'this' used in React when something is held in 'state' or 'setSet'. The the value of it is declared earlier on, so that allows you to refer to the method or key/value pair with 'this' in front of it. The 'this' helps you not have to repeat declaration. I like to think of using 'this' like if the Class was a huge house, and if you wanted to call someone into another room of the house without walking all the way to that room, you would call them on your cellphone. Although, using 'this' in a file that lives in the components would be known as calling ('this'.props), to me it is like having a guest house because you can still call on it from another file that is associated with the main JavaScript file. Either way, the 'this' is used to call on something that was previously given a value that is held within the state or method of the same React App.
 
+  Researched answer:
 
+  What I explained was the use of 'this' in React, but even in React the main point is the 'this' is used in Class structures. React just uses a big class and so that was the easiest reference for me. The 'this' is the cellphones of Class houses. A way to call on specific information. The rooms of the house, are called objects.
```

Comment:
Awesome analogy as always!

Reply:
This analogy actually helped like 5 people in class understand Classes in React better! Hence my uplifted mood last week.